composer.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "spryker-eco/oauth-azure",
+  "type": "library",
+  "description": "OauthAzure module",
+  "license": "MIT",
+  "require": {
+    "league/oauth2-client": "^2.0.0",
+    "php": ">=7.3",
+    "spryker/kernel": "^3.30.0",
+    "spryker/log": "^3.7.0",
```

Comment:
What makes this version of `spryker/log` required? Why not 3.0.0?

Reply:
Because the current version has a memory leak issue fix.